pkg/util/util.go

```diff
@@ -83,6 +88,31 @@ func IsResourceIncluded(resource string, backup *velerov1.Backup) bool {
 	return false
 }
 
+func IsResourceExcluded(resource string, backup *velerov1.Backup) bool {
```

Comment:
string resource means resource kind or a resource name

Reply:
Good point. Changing to `resourceKind`.